.github/workflows/release.yml

```diff
@@ -68,82 +66,117 @@ jobs:
         run: |
-          ls -lrt images/DAppNode-debian-bookworm-amd64.iso
+          ls -lrt images/Dappnode-debian-*.iso
 
-      # Set new name for the release asset
-      - name: Set DAppNode attended ISO name
+      - name: Set Debian Dappnode attended ISO name
         run: |
-          cp ./images/DAppNode-debian-bookworm-amd64.iso DAppNode-${CORE_VERSION}-debian-bookworm-amd64.iso
+          file=$(ls images/Dappnode-debian-*.iso)
+          attended_filename="${file/images\/Dappnode-/Dappnode-${CORE_VERSION}-}"
+          mv "$file" "$attended_filename"
 
-      # ISO UNATTENDED
-      - name: Build unattended
+      - name: Build Debian unattended
         run: |
           sed -i -e "/UNATTENDED/s/false/true/" docker-compose.yml
-          docker-compose build
-          docker-compose up
+          docker-compose up --build
 
-      # Verify ISO unattended was created
-      - name: Check iso unattended
+      - name: Check Debian ISO unattended
         run: |
-          ls -lrt images/DAppNode-debian-bookworm-amd64.iso
+          ls -lrt images/Dappnode-debian-*.iso
 
-      # Set new name for the release asset
-      - name: Set DAppNode unttended ISO name
+      - name: Set Dappnode unattended ISO name
         run: |
-          cp ./images/DAppNode-debian-bookworm-amd64.iso DAppNode-${CORE_VERSION}-debian-bookworm-amd64-unattended.iso
+          file=$(ls images/Dappnode-debian-*.iso)
+          core_filename="${file/images\/Dappnode-/Dappnode-${CORE_VERSION}-}"
+          unattended_filename="${core_filename/%.iso/-unattended.iso}"
+          mv "$file" "$unattended_filename"
+
+      - name: Build Ubuntu attended
+        run: |
+          sed -i -e "/BASE_OS/s/debian/ubuntu/" docker-compose.yml
+          sed -i -e "/UNATTENDED/s/true/false/" docker-compose.yml 
+          docker-compose up --build
+
+      - name: Check Ubuntu ISO attended
+        run: |
+          ls -lrt images/Dappnode-ubuntu-*.iso
+
+      - name: Set Ubuntu Dappnode attended ISO name
+        run: |
+          file=$(ls images/Dappnode-ubuntu-*.iso)
+          attended_filename="${file/images\/Dappnode-/Dappnode-${CORE_VERSION}-}"
+          mv "$file" "$attended_filename"
+
+      - name: Build Ubuntu unattended
+        run: |
+          sed -i -e "/BASE_OS/s/ubuntu/debian/" docker-compose.yml
+          sed -i -e "/UNATTENDED/s/false/true/" docker-compose.yml
+          docker-compose up --build
+
+      - name: Check Ubuntu ISO unattended
+        run: |
+          ls -lrt images/Dappnode-ubuntu-*.iso
+
+      - name: Set Ubuntu Dappnode unattended ISO name
+        run: |
+          file=$(ls images/Dappnode-ubuntu-*.iso)
+          core_filename="${file/images\/Dappnode-/Dappnode-${CORE_VERSION}-}"
+          unattended_filename="${core_filename/%.iso/-unattended.iso}"
+          mv "$file" "$unattended_filename"
 
-      # Create profile.sh script (not able to set dot (.) before the name in the gh release asset)
```

Comment:
why removing this comment? its critical if someone attempts to add an artifact that starts with dot